test/utils/expected-files.js

```diff
@@ -22,18 +22,17 @@ const expectedFiles = {
     server: {
         common: [
             `${SERVER_MAIN_RES_DIR}application.properties`,
-            `${SERVER_MAIN_RES_DIR}jwt/privateKey.pem`,
-            `${SERVER_MAIN_RES_DIR}META-INF/resources/publicKey.pem`,
             `${SERVER_MAIN_SRC_DIR}com/mycompany/myapp/config/Constants.java`,
             `${SERVER_MAIN_SRC_DIR}com/mycompany/myapp/config/JHipsterProperties.java`,
             `${SERVER_MAIN_SRC_DIR}com/mycompany/myapp/config/JsonbConfiguration.java`,
-            `${SERVER_MAIN_SRC_DIR}com/mycompany/myapp/security/jwt/TokenProvider.java`,
             `${SERVER_MAIN_SRC_DIR}com/mycompany/myapp/web/rest/errors/BadRequestAlertException.java`,
+            // `${SERVER_MAIN_SRC_DIR}com/mycompany/myapp/web/rest/AccountResource.java`,
```

Comment:
remove?

Reply:
Yes It was a reminder to create an issue actually :)